Bottom: f99ce383237cf2454c45e692749b9663f2db6301
Top:    72201536adf1651e111a60e6cb0ce96f44a7fbc6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 02:01:55 +0100

Refresh of fix-test-suite-harder

---

diff --git a/tests/golden/hid_gcode1/Makefile.am b/tests/golden/hid_gcode1/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode1/Makefile.am
+++ b/tests/golden/hid_gcode1/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode10/Makefile.am b/tests/golden/hid_gcode10/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode10/Makefile.am
+++ b/tests/golden/hid_gcode10/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode11/Makefile.am b/tests/golden/hid_gcode11/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode11/Makefile.am
+++ b/tests/golden/hid_gcode11/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode2/Makefile.am b/tests/golden/hid_gcode2/Makefile.am
index a757c9e..a771564 100644
--- a/tests/golden/hid_gcode2/Makefile.am
+++ b/tests/golden/hid_gcode2/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	out.back.cnc \
+	out.bottom.cnc \
 	out.drill.cnc \
-	out.front.cnc
+	out.top.cnc
diff --git a/tests/golden/hid_gcode3/Makefile.am b/tests/golden/hid_gcode3/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode3/Makefile.am
+++ b/tests/golden/hid_gcode3/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode4/Makefile.am b/tests/golden/hid_gcode4/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode4/Makefile.am
+++ b/tests/golden/hid_gcode4/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode5/Makefile.am b/tests/golden/hid_gcode5/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode5/Makefile.am
+++ b/tests/golden/hid_gcode5/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode6/Makefile.am b/tests/golden/hid_gcode6/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode6/Makefile.am
+++ b/tests/golden/hid_gcode6/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode7/Makefile.am b/tests/golden/hid_gcode7/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode7/Makefile.am
+++ b/tests/golden/hid_gcode7/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode8/Makefile.am b/tests/golden/hid_gcode8/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode8/Makefile.am
+++ b/tests/golden/hid_gcode8/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gcode9/Makefile.am b/tests/golden/hid_gcode9/Makefile.am
index b071e19..b07c896 100644
--- a/tests/golden/hid_gcode9/Makefile.am
+++ b/tests/golden/hid_gcode9/Makefile.am
@@ -1,6 +1,6 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gcode_oneline.gcode.back.cnc \
+	gcode_oneline.gcode.bottom.cnc \
 	gcode_oneline.gcode.drill.cnc \
-	gcode_oneline.gcode.front.cnc
+	gcode_oneline.gcode.top.cnc
diff --git a/tests/golden/hid_gerber1/Makefile.am b/tests/golden/hid_gerber1/Makefile.am
index 9d32ff3..67f1c2a 100644
--- a/tests/golden/hid_gerber1/Makefile.am
+++ b/tests/golden/hid_gerber1/Makefile.am
@@ -1,7 +1,7 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	gerber_oneline.back.gbr \
+	gerber_oneline.bottom.gbr \
 	gerber_oneline.fab.gbr \
-	gerber_oneline.front.gbr \
+	gerber_oneline.top.gbr \
 	gerber_oneline.plated-drill.cnc
diff --git a/tests/golden/hid_gerber2/Makefile.am b/tests/golden/hid_gerber2/Makefile.am
index e07c8b4..4553101 100644
--- a/tests/golden/hid_gerber2/Makefile.am
+++ b/tests/golden/hid_gerber2/Makefile.am
@@ -1,7 +1,7 @@
 ## -*- makefile -*-
 
 EXTRA_DIST= \
-	out.back.gbr \
+	out.bottom.gbr \
 	out.fab.gbr \
-	out.front.gbr \
+	out.top.gbr \
 	out.plated-drill.cnc
